.rubocop.yml

```diff
@@ -160,3 +160,15 @@ Style/Semicolon:
 # Prefer Foo.method over Foo::method
 Style/ColonMethodCall:
   Enabled: true
+
+Style/SymbolProc:
+  Enabled: true
```

Comment:
I prefer Non Symbol Proc, because it is much readable~

Reply:
I agree, take this out unless it's also faster. Let's scope this change to only performance.

Reply:
Benchmark/ips says that they're about the same 

```

require 'benchmark/ips'

array = ["foo", "bar", "baz"]


Benchmark.ips do |x|
  x.report("symbol proc ") { array.each(&:upcase) }
  x.report("regular proc") { array.each {|x| x.upcase } }
  x.compare!
end
```